db.go

```diff
@@ -21,46 +20,11 @@ var (
 
 // DB is a client for the SurrealDB database that holds are websocket connection.
 type DB struct {
-	ws *websocket.WebSocket
-}
-
-// Option is a struct that holds options for the SurrealDB client.
-type Option struct {
-	WsOption websocket.Option
-}
-
-// WithTimeout sets the timeout for requests, default timeout is 30 seconds
-func WithTimeout(timeout time.Duration) Option {
-	return Option{
-		WsOption: func(ws *websocket.WebSocket) error {
-			ws.Timeout = timeout
-			return nil
-		},
-	}
-}
-
-// UseWriteCompression enables or disables write compression for internal websocket client
-func UseWriteCompression(useWriteCompression bool) Option {
-	return Option{
-		WsOption: func(ws *websocket.WebSocket) error {
-			ws.Conn.EnableWriteCompression(useWriteCompression)
-			return nil
-		},
-	}
+	ws iwebsocket.IWebSocket
```

Comment:
In golang there is no convention to prefix interfaces with `I`. This goes for packages and the interface itself. Instead I would recommend package `websocket` and interface `WebSocket`. Then implementations can be different to `WebSocket`.

Reply:
done.